### ISSUE TITLE
Missing sweet_xml vulnerability?

### ISSUE THREAD

**Issue by Cohen-Carlisle** [nccgroup/sobelow#66]
A teammate suggested that we run [mix_audit](https://github.com/mirego/mix_audit) in addition to sobelow. I thought it was unnecessary since sobelow also does patch level verification but to my surprise mix_audit reported [CVE-2019-15160](https://nvd.nist.gov/vuln/detail/CVE-2019-15160) as a vulnerability while sobelow did not. MixAudit actually gets its vulnerability info from [dependabot/elixir-security-advisories](https://github.com/dependabot/elixir-security-advisories).

TL;DR I'm wondering if sobelow wants to reach parity with the elixir-security-advisories database or if it has excluded any vulnerabilities for specific reasons.

**Comment by GriffinMB** [comment]
Hey, great question!

Although the initial intention was to check all dependencies for vulnerabilities, a couple of years ago I narrowed the scope to only core Phoenix libraries - ie Phoenix itself, Plug, etc. A combination of `mix hex.audit` and MixAudit are probably the ideal ways to track dependency concerns, while Sobelow is more strictly static analysis focused.

The docs should definitely be updated to indicate that MixAudit is more full-featured for vulnerability tracking. Would love a PR, otherwise I'll think about the language and make the change myself.
